subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/SmokeTestGradleRunner.groovy

```diff
@@ -106,7 +106,8 @@ class SmokeTestGradleRunner extends GradleRunner {
         return this
     }
 
-    SmokeTestGradleRunner ignoreDeprecationWarnings() {
+    @SuppressWarnings('unused')
```

Comment:
Why is this marked as unused?

Reply:
Because the parameter is unused. I changed it around to log the message instead.